pom.xml

```diff
@@ -33,6 +33,7 @@ not datatype, data format, or JAX-RS provider modules.
     <module>mrbean</module>
     <module>osgi</module>
     <module>paranamer</module>
+    <module>subtype</module>
```

Comment:
I'll probably want to change the name, "subtype" is too generic. But let me think about better name for a while...

Reply:
It's a difficult thing for me to give it a name...

Reply:
True🥲🥲